src/sidebar/components/MarkdownView.tsx

```diff
@@ -161,7 +161,7 @@ export default function MarkdownView(props: MarkdownViewProps) {
             open={!!popoverContent}
             onClose={() => setPopoverContentAfterDelay(null)}
             anchorElementRef={mentionsPopoverAnchorRef}
-            classes="px-3 py-2"
+            classes="px-3 py-2 !max-w-[75%]"
```

Comment:
Limit the size of the popover to a maximum of 75% of the sidebar, to let it breathe.